Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,253 @@
+    else:
+        r = requests.request(method,
+                             full_url,
+                             headers=headers,
+                             json=data)
+    if (r.status_code == 404):
+        return_error("Issue {} not found.".format(URL_SUFFIX[URL_SUFFIX.rfind("/") + 1:]))
+    if (r.status_code == 422):
+        return_error("One of the users {} is unassaignable.".format(data['assignees']))
+    if r.status_code not in [200, 201]:
+        return_error('Error in API call [{}] -{}. headers: {} \n url: {}\ndata: {}'.format(
+            r.status_code, r.reason, headers, API + URL_SUFFIX, data))
+
+    return r.json()
+
+
+'''COMMAND FUNCTIONS'''
+
+
+def list_all_issues_command():
+    LOG("GitHub: Fetching open issues for repository: {}/{}".format(OWNER, REPO))
+    context_entries = []
+    res = http_request('GET', "/issues")
+    for issue in res:
+        context_entries.append({
+            'ID': issue.get('number'),
+            'Title': issue.get('title'),
+            'Body': issue.get('body'),
+            'Assignees': [assignee.get('login') for assignee in issue.get('assignees')],
+            'Labels': [label.get('name') for label in issue.get('labels')]
+        })
+    HR = tableToMarkdown("Open issues", context_entries)
+    EC = {'GitHub.Issue(val.ID && val.ID == obj.ID)': context_entries}
+    return_outputs(HR, EC, res)
+
+
+def create_issue_command():
+    title = demisto.args().get('title')
+    body = demisto.args().get('body')
+    assignees = argToList(demisto.args().get('assignees'))
+    labels = argToList(demisto.args().get('labels'))
+
+    LOG("GitHub: Creating a new issue with title %s" % (title))
+
+    create_issue(title, body, assignees, labels)
+
+
+def create_issue(title, body, assignees, labels):
+    data = {'title': title}
+    if (body is not None):
+        data['body'] = body
+    if (len(assignees) > 0):
+        ##check assignees
+        data['assignees'] = assignees
+    if (len(labels) > 0):
+        data['labels'] = labels
+    res = http_request('POST', "/issues", data)
+
+    context_entries = {
+        'ID': res.get('number'),
+        'Title': res.get('title'),
+        'Body': res.get('body'),
+        'Assignees': [assignee.get('login') for assignee in res.get('assignees')],
+        'Labels': [label.get('name') for label in res.get('labels')]
+    }
+    HR = tableToMarkdown("Created issue", context_entries,
+                         ["ID", "Title", "Body", "Assignees", "Labels"])
+    EC = {'GitHub.Issue(val.ID && val.ID == obj.ID)': context_entries}
+    return_outputs(HR, EC, res)
+
+
+def close_issue_command():
+    issues = argToList(str(demisto.args().get('issue_number')))
+    s = ""
+    for issue in issues:
+        close_issue(issue)
+        s = s + str(issue) + ", "
+    demisto.results("Closed issues number: %s" % s[:-2])
+
+
+def close_issue(issue_number):
+    data = {"state": "closed"}
+    http_request("POST", "/issues/" + str(issue_number), data)
```

Comment:
Technically yes. This would give a 404 status which is taken care of explicitly in the http requests funciton:
    if (r.status_code == 404):
        return_error("Issue {} not found.".format(url_suffix[url_suffix.rfind("/") + 1:]))